dlfcn_freebsd.go

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2022 The Ebitengine Authors
+
+package purego
+
+const (
```

Comment:
Do you have a link that shows these constants in C?

Reply:
They are defined in https://github.com/freebsd/freebsd-src/blob/main/lib/csu/common/csu_common.h
The difference with FreeBSD is that the CSU components are not part of libc, and unfortunately libc.so depends on them to exist in the compiled executables.

Reply:
oops https://github.com/freebsd/freebsd-src/blob/main/include/dlfcn.h

Reply:
Great! Can you leave this link as a comment in the code that way it's documented